tests/signals/tests.py

```diff
@@ -258,6 +258,19 @@ def __call__(self, signal, sender, **kwargs):
         self.assertTrue(b._run)
         self.assertEqual(signals.post_save.receivers, [])
 
+    @mock.patch('weakref.ref')
+    def test_lazy_model_signal(self, ref):
+        def callback(sender, args, **kwargs):
+            pass
+        signals.pre_init.connect(callback)
+        signals.pre_init.disconnect(callback)
+        self.assertTrue(ref.called)
```

Comment:
From looking at the code the call could differ from Python 2 to 3 and is really an implementation detail which is not worth testing after all.

My initial reflexion was more about the fact `assert_not_called()` was used below instead of `self.assertFalse(ref.called)` but now I realize there's no `assert_called()` method.

LGTM